### ISSUE TITLE
F5-TTS Integration

### ISSUE THREAD

**Issue by nityanandmathur** [huggingface/diffusers#10043]
### Model/Pipeline/Scheduler description

F5-TTS is a fully non-autoregressive text-to-speech system based on flow matching with Diffusion Transformer (DiT).
It has excellent voice cloning capabilities, and audio generation is of quite high quality.

### Open source status

- [X] The model implementation is available.
- [X] The model weights are available (Only relevant if addition is not a scheduler).

### Provide useful links for the implementation

Paper - https://arxiv.org/abs/2410.06885
Code - https://github.com/SWivid/F5-TTS?tab=readme-ov-file
Weights - https://huggingface.co/SWivid/F5-TTS

Author - @SWivid

**Comment by sayakpaul** [comment]
@ylacombe is this worth integrating?

**Comment by ylacombe** [comment]
It is @sayakpaul, F5-TTS is one of the best TTS models out here. The only caveat is its license (NC) but I think it's worth the shot. Would you like to take of this @nityanandmathur ?

**Comment by tuanh123789** [comment]
hi @ylacombe can i try

**Comment by ylacombe** [comment]
Hey @tuanh123789 , feel free!

**Comment by ayushtues** [comment]
@sayakpaul @ylacombe can I pick this up? This has had a lot of impact and follow ups, imo still worth adding

**Comment by sayakpaul** [comment]
Sure.

**Comment by asomoza** [comment]
I tested this model and it's really good, maybe we should give it a chance if it can be integrated with diffusers.

**Comment by ayushtues** [comment]
Yes, will work over the weekend on this, would be fun to integrate this!

**Comment by ayushtues** [comment]
Started #11958 , tracking progress here: https://satisfying-writer-e7e.notion.site/F5-TTS-HF-integration-235ee5eabbf88019a3b7ce6659dfabc0

**Comment by b-sai** [comment]
Is this issue still open? I'd like to contribute here

**Comment by ktk-07** [comment]
@DN6 @asomoza It seems like no one is actively working on this issue and help is wanted?  I really want this model to be able to work on this repo and I will be more than willing to help with this. Are you able to assign it to me?
